MuonAnalysis/MuonAssociators/src/MatcherUsingTracksAlgorithm.cc

```diff
@@ -182,6 +182,7 @@ bool MatcherUsingTracksAlgorithm::match(const reco::Candidate &c1,
         }
       }
     }
+      [[fallthrough]];
```

Comment:
@santocch @mrodozov a ```break``` would get the same effect here I would say, anyway it does not seem incorrect to me

Reply:
I agree. The logic would be identical... I'm not sure which one would be better or what difference would a break introduce... probably none so let's keep this one.